backend/game/shop/shop.py

```diff
@@ -0,0 +1,63 @@
+import json
+import random
+
+from typing import List, Dict, TextIO, Tuple
+from math import exp, floor
+
+class Shop:
+
+    path_to_units_file: str = './../data/units.json'
```

Comment:
To pójdzie na Windowsie, wszyscy mają Linuxa...?

Reply:
Relatywność na każdym systemie zadziała tak samo.
Bardziej się obawiam, czy z kontekstu odpalenia aplikacji (inny working directory niż relatywna ścieżka od tego pliku) odpali to poprawnie - nie przetestowałem tego ze strony aplikacji odpalanej zapewne w folderze projektu.

Reply:
Jestem za dzieleniem :+1:

Reply:
A, to prawdopodobnie nie zadziała jeśli odpali to server